tests/test_tensor.py

```diff
@@ -106,30 +109,39 @@ def test_zeros(backend):
 
 def test_broadcasting(backend):
     tb = pyhf.tensorlib
-    assert list(
-        map(
-            tb.tolist,
-            tb.simple_broadcast(
-                tb.astensor([1, 1, 1]), tb.astensor([2]), tb.astensor([3, 3, 3])
-            ),
+    assert (
```

Comment:
These blocks seem like they're really going to stay as is for the time being given things like https://github.com/psf/black/issues/1629